### ISSUE TITLE
FreqGroup class seems to be in pandas._libs.tslibs.dtypes

### ISSUE THREAD

**Issue by garciapd** [bsolomon1124/pyfinance#18]
I'm trying to use PandasRollingOLS but the class FreqGroup is not found, neither the get_freq_code.

I have looked at Panda and seems that the lib is in:

``from pandas._libs.tslibs.dtypes import FreqGroup`` instead of ``from pandas._libs.tslibs.frequencies import FreqGroup, get_freq_code``

at utils.py line 80

If I do this change the lib works, just telling in case you want to change it, i'm not doing a PR because I don't know if that can affect something else, did really quick.

Cheers

**Comment by vilasfe** [comment]
I did a little digging and it looks like pandas changed the API for FreqGroup between 1.0.x and 1.1.x. So it is a relatively recent change. There could probably be a PR based on something like

> import pandas as pd
> if pd._version.get_versions()['version'].startswith('0.') or pd._version.get_versions()['version'].startswith('1.0'):
>    >  from pandas._libs.tslibs.frequencies import FreqGroup, get_freq_code  

> else:
> >    from pandas._libs.tslibs.dtypes import FreqGroup
`

NOTE: I have not tested the above code, it is just speculation at this point.

**Comment by JS00N** [comment]
This seems like pretty significant bug as it prevents pyfinance from being imported.

`FreqGroup` and `get_freq_code` seem to be used exclusively in the `get_anlz_factor` function. As noted by garciapd, `FreqGroup` has been moved to `pandas._libs.tslibs.dtypes`. I have searched the source code of pandas 1.1.2 and `get_freq_code` no longer exists with the functionality implemented in pyfinance. [Attached ](https://github.com/bsolomon1124/pyfinance/files/5324166/pyfinance_fix.py.txt) is a script that defines a function `get_freq_code` which has the proper functionality. I have tested it successfully on all the test cases in the `get_anlz_factor` docstring. I did have to modify the expected exception in  `get_anlz_factor` (line 56).

Edit: A pull request has been created.
